src/scp/test/SCPTests.cpp

```diff
@@ -2452,6 +2510,53 @@ TEST_CASE("ballot protocol core3", "[scp][ballotprotocol]")
             }
         }
     }
+    SECTION("prepared A1 with timeout")
```

Comment:
maybe i'm missing something, but the name seems confusing: at the end of the test, B2 was prepared

Reply:
section starts with `prepared A1` and then timeouts